File: android studio/KLIMOV/DAY07/app/src/main/java/com/example/day_7_5/MainActivity.java
package com.example.day_7_5;

import androidx.appcompat.app.AppCompatActivity;

import android.content.pm.ActivityInfo;
import android.content.res.Configuration;
import android.os.Bundle;
import android.view.OrientationEventListener;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity
{
    TextView textView;
    Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        button = findViewById(R.id.button);
        textView = findViewById(R.id.textView);

        button.setOnClickListener(new View.OnClickListener()
        {

            public void onClick(View v)
            {
                textView.setText("Pidorok");
            }
        });
    }


}

File: android studio/KLIMOV/DAY15K/app/src/main/java/com/example/day15k/MainActivity.kt
package com.example.day15k
import android.os.Bundle
import android.view.Gravity
import android.widget.Button
import android.widget.EditText
import android.widget.ImageView
import android.widget.LinearLayout
import android.widget.RadioButton
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity()
{

    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val meterRadioButton: RadioButton = findViewById(R.id.radio_button_meter)
        val inputEditText: EditText = findViewById(R.id.editText)

        val button: Button = findViewById(R.id.button_converter)

        button.setOnClickListener {
            if (inputEditText.text.isEmpty())
            {
                Toast.makeText(applicationContext, "Введите длину кота", Toast.LENGTH_SHORT).show()
            }
            else
            {
                val inputValue = inputEditText.text.toString().toFloat()
                if (meterRadioButton.isChecked)
                    inputEditText.setText(convertParrotToMeter(inputValue).toString())
                else
                    inputEditText.setText(convertMeterToParrot(inputValue).toString())
            }
        }
    }

    private fun convertParrotToMeter(parrot: Float): Float = (parrot / 7.6).toFloat()

    private fun convertMeterToParrot(meter: Float): Float = (meter * 7.6).toFloat()
}